src/components/sections/DemographicsSection/DemographicsSection.js

```diff
@@ -39,20 +41,46 @@ const sexOptions = [
 ];
 
 function DemographicsSection() {
-  const { form } = useForm();
+  const { values, setFieldValue } = useFormikContext();
+
+  const handleBirthdayChange = useCallback(
+    date => {
+      batch(() => {
+        setFieldValue('dateOfBirth', date);
+        if (!isNaN(date)) {
+          const now = new Date();
+          const dayDiff = differenceInDays(now, date);
+          const monthDiff = differenceInMonths(now, date);
```

Comment:
should there be a `const yearDiff = differenceInYears(now, date)` for consistency?

Reply:
Yep! Will fix. :)